portafly/src/tests/components/shared/data-list/DataListContext.test.tsx

```diff
@@ -54,3 +56,59 @@ describe('DataListContext: useDataFilters', () => {
     expect(filters.innerHTML).toBe('')
   })
 })
+
+describe('DataListContext: DataReducer', () => {
+  const ExampleComponent = () => {
+    const { data, setData } = useDataListData()
```

Comment:
```suggestion
    const { data, setData } = useDataListData<IDeveloperAccount[]>()
```

Reply:
Doesn't apply it seems.

Reply:
It's related to https://github.com/3scale/porta/pull/2036#discussion_r452087240